x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/metrics.tsx

```diff
@@ -94,16 +102,14 @@ function buildMetricOperation<T extends MetricColumn<string>>({
         sourceField: field.name,
       };
     },
-    toEsAggsConfig: (column, columnId, _indexPattern) => ({
-      id: columnId,
-      enabled: true,
-      type: column.operationType,
-      schema: 'metric',
-      params: {
+    toEsAggsFn: (column, columnId, _indexPattern) => {
+      return buildExpressionFunction(typeToFn[type], {
+        id: columnId,
+        enabled: true,
+        schema: 'metric',
         field: column.sourceField,
-        missing: 0,
```

Comment:
I don't believe `missing` is a valid param for any of these agg types so I removed it; please LMK if you know otherwise.